Neo.ConsoleService/ConsoleServiceBase.cs

```
@@ -540,8 +529,8 @@ public void Run(string[] args)
                         RedirectStandardOutput = true,
                         UseShellExecute = false
                     });
-                    process.WaitForExit();
-                    Console.Write(process.StandardOutput.ReadToEnd());
+                    process?.WaitForExit();
```

I think we should throw the exception here instead of ignoring it. The easiest way is to remove the question mark.